### ISSUE TITLE
Document long running (LR) queries grace period 

### ISSUE THREAD

**Issue by aashishkohli** [ClickHouse/clickhouse-docs#1698]
**Context**
Currently we:
1. Wait 1 hour for LR queries before we interrupt them for **server upgrades**. 
2. We never interrupt queries for **autoscaling** if they have been running for > 70 mins. We are looking to change this behavior so that we will interrupt after 1 hr and be consistent with server upgrades. We will do so for select queries [soon](https://github.com/ClickHouse/data-plane-application/pull/6987), and will do the same for insert queries after we launch MBB.
More context here: https://github.com/ClickHouse/data-plane-application/issues/6798

This issue is a placeholder for now, but we should share this guidance with customers via docs, and we also link to it from appropriate places in the UI. One potential issue is that we have changed the LR queries grace period a few times in the past, so we should be aligned on this before we make it public. In documenting this we should also provide guidance to customers about what they do do for LR inserts > 1 hr, such as using clickpipes etc.

cc: @krithika-ch

**Comment by johnnymatthews** [comment]
I'm assuming this is specific to Cloud, correct?

**Comment by aashishkohli** [comment]
Yes, @johnnymatthews - this is for cloud as we are in charge of restarting servers. In self-installed mode customers can do what they want.

**Comment by gingerwizard** [comment]
@aashishkohli do we want to document this?
